globals.go

```diff
@@ -377,6 +377,10 @@ const (
 	// HostServicePasswordExpansion is the expansion for the service password that is stored on the host,
 	// and is meant to be set as a private variable so that it will be redacted in all logs.
 	HostServicePasswordExpansion = "host_service_password"
+
+	// RedactedWebhookSecretsValue is the value that is shown in the REST API and UI for a redacted
+	// webhook header authorization value.
+	RedactedWebhookSecretsValue = "<REDACTED>"
```

Comment:
I imagine we could likely reuse this string to standardize as the generic redaction placeholder. Could we just call this `RedactedValue` or something like that?